README.md

```diff
@@ -228,7 +229,7 @@ def new(...) do
 end
 ```
 
-Passing directly to `get`/`post`/etc.
+Passing directly to `get` or etc.
```

Comment:
Since the examples after the statement have two different function calls, I've updated to:

```
  232 Passing directly to request functions such as `MyClient.get/3` or `Tesla.get/3`.                      
  233                                                                                                       
  234 ```elixir                                                                                             
  235 MyClient.get("/", opts: [adapter: [recv_timeout: 30_000]])                                            
  236 Tesla.get(client, "/", opts: [adapter: [recv_timeout: 30_000]])                                       
  237 ```  
```